Strategy11/formidable-forms
# Introduce bottom banner for keyboard command guidance

Introduce a bottom banner to educate users about using keyboard commands for selecting multiple fields.

## Related Issue:
[Add way to let users know about keyboard commands](https://github.com/Strategy11/formidable-pro/issues/3138)

## QA URL
https://qa.formidableforms.com/sherv8/wp-admin/admin.php?page=formidable&frm_action=edit&id=24

## Testing Instruction:
1. Open the form builder.
2. Select a field to trigger the bottom banner.
3. Confirm the bottom banner appears with guidance on keyboard commands.
4. Ensure the banner can be manually closed by clicking the 'x' button.

## Output Video
https://www.loom.com/share/cd3037e079954b7e952991bcccd41a6a?sid=f42be2ec-3840-47a9-9434-5daedd7da97a
